### DISCUSSION TITLE
Great hardware solution to use octo4a

### DISCUSSION THREAD

**Discussion by aydinakcasu** [feelfreelinux/octo4a#180] [General]
If you want to use octoprint, but can’t find a cheap raspberry pi?

I came up with a great solution.
- Use octo4a, octoprint for android phones (https://github.com/feelfreelinux/octo4a)
- android phones with micro usbs often have an issue with being plugged in, and still be chargeable, so instead….
- buy a “Blue view 2” phone (https://g.co/kgs/jWAbUQ).  Why? It’s $30. I bought mine at riteaid, but you can buy them everywhere. It has a usb c port, that does allow to be used and changed at the same time.  (This is a prepaid phone, just don’t put in the SIM card and don’t activate it.  Just use it as an android device over Wi-Fi) (the phone runs android 10, is plenty fast, and has an micro sd card slot)
- you need a cable, like a “USB-C® to HDMI™ & USB™ 3.0 with Power Delivery” cable, https://en.j5create.com/products/jca379, for $30-40 (there are $10 cables that do the same thing on Amazon apparently).

So for under $70 ($40, with the cheaper cable), you can have octoprint running, Plus you can use the phone camera.  

It works great!!!

**Comment by Joerideman** [comment]
That is pretty cool. Even in the Netherlands I can find these.

Could you tell me what baudrates you can use? I have an old sony that doesn't want to play at 230400 or 250000 bit/s even though my printer has no problem with it.
